recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -1,25 +1,27 @@
 <template>
-  <div :class="['dt-feed-item-pill--border', borderClass, wrapperClass]">
-    <div class="d-p8 d-bgc-secondary">
+  <div :class="['dt-recipe-feed-item-pill__border', borderClass, wrapperClass]">
+    <div class="dt-recipe-feed-item-pill__wrapper">
       <dt-collapsible :open="expanded">
         <template #anchor>
           <button
+            data-qa="dt-recipe-feed-item-pill"
             :aria-label="ariaLabel"
-            data-qa="dt-feed-item-pill"
-            :class="['d-baw0 d-bgc-moderate d-bar-pill d-w100p d-ta-left d-btn--circle', toggleableClass, buttonClass]"
+            :class="['dt-recipe-feed-item-pill__button', toggleableClass, buttonClass]"
             @focusin="hover = true"
             @focusout="hover = false"
             @mouseenter="hover = true"
             @mouseleave="hover = false"
             @click="onClick"
           >
-            <dt-item-layout class="d-w100p d-p8">
-              <span class="d-fw-bold">{{ title }}</span>
+            <dt-item-layout class="dt-recipe-feed-item-pill__layout">
+              <slot name="title">
```

Comment:
@AlexLamDialpad added this so we can use the slot, in case the title needs to be handled differently.